homeassistant/components/risco/binary_sensor.py

```diff
@@ -3,13 +3,25 @@
     DEVICE_CLASS_MOTION,
     BinarySensorEntity,
 )
+from homeassistant.helpers import config_validation as cv, entity_platform
 
 from .const import DATA_COORDINATOR, DOMAIN
 from .entity import RiscoEntity
 
+SERVICE_BYPASS_ZONE = "bypass_zone"
```

Comment:
you could consider a bypass "switch" rather than services since its just a toggle essentially

Reply:
I thought about this, but:
* This is a rare operations
* You can have dozens, or even hundreds, of zones

Given that, it seemed that adding a switch per zone, and figuring out how to map between the two in automations (e.g. if you want to bypass a zone based on it triggering in some situation), isn't the best approach.